randovania/games/fusion/generator/bootstrap.py

```diff
@@ -35,6 +37,21 @@ def _get_enabled_misc_resources(
         enabled_resources.add("GeneratorHack")
         return enabled_resources
 
+    def _damage_reduction(self, db: ResourceDatabase, current_resources: ResourceCollection) -> float:
+        num_suits = sum(
+            (1 if current_resources[db.get_item_by_name(suit)] else 0) for suit in ("Varia Suit", "Gravity Suit")
+        )
+        dr = 1.0
+        if num_suits == 1:
+            dr = 0.9
+        elif num_suits >= 2:
+            dr = 0.8
```

Comment:
iirc this isnt correct. varia always gives the first set of DR and Grav gives the second. regardless of amount of suits. may need someone to confirm that.

Reply:
Anty confirmed this a long time ago, that suit damage is progressive

Reply:
> anyway enemy damage reduction can be 30%/10%, 60%/30%, 80%/10%, and 90%/80%
left being single suit, right being both